yt_dlp/networking/_curlcffi.py

```diff
@@ -31,9 +32,9 @@
 
 curl_cffi_version = tuple(map(int, re.split(r'[^\d]+', curl_cffi.__version__)[:3]))
 
-if curl_cffi_version != (0, 5, 10) and not ((0, 7, 0) <= curl_cffi_version < (0, 7, 2)):
+if curl_cffi_version != (0, 5, 10) and not (0, 10) <= curl_cffi_version:
     curl_cffi._yt_dlp__version = f'{curl_cffi.__version__} (unsupported)'
-    raise ImportError('Only curl_cffi versions 0.5.10, 0.7.0 and 0.7.1 are supported')
+    raise ImportError('Only curl_cffi versions 0.5.10 and 0.10+ are supported')
```

Comment:
since 0.11 would raise this too?
```suggestion
    raise ImportError('Only curl_cffi versions 0.5.10 and 0.10.x are supported')
```